app/views/layouts/admin/main.html.slim

```diff
@@ -25,6 +25,7 @@ html lang="en-US"
           dateFormats: #{json(I18n.t("date.moment_formats"))},
           locale: #{json(I18n.locale)}
         });
+        window.popoverView = new MS.Views.PopoverView({el: "body"});
```

Comment:
Separated this out for separation of concerns and b/c we need to pass it to QuestionsView.